help/sources/tutorials/ui/create/advertising/linkedin-ads.md

```diff
@@ -0,0 +1,150 @@
+
+You can select the appropriate category from the catalog on the left-hand side of your screen. Alternatively, you can find the specific source you wish to work with using the search option.
+
+Under the *Advertising* category, select *[!DNL LinkedIn Ads]*, and then select **[!UICONTROL Add data]**.
+
+![Platform UI screenshot for catalog with LinkedIn Ads card](../../../../images/tutorials/create/linkedin-ads/catalog.png?lang=en)
+
+The **[!UICONTROL Connect LinkedIn Ads account]** page appears. On this page, you can either use new credentials or existing credentials.
+
+### Existing account
+
+To use an existing account, select the [!DNL LinkedIn Ads] account you want to create a new dataflow with, then select **[!UICONTROL Next]** to proceed.
+
+![Platform UI screenshot for Connect LinkedIn Ads account with an existing account](../../../../images/tutorials/create/linkedin-ads/existing.png?lang=en)
+
+### New account
+
+If you are creating a new account, select **[!UICONTROL New account]**, and then provide a name, an optional description, and your credentials. When finished, select **[!UICONTROL Connect to source]** and then allow some time for the new connection to establish.
+
+![Platform UI screenshot for Connect LinkedIn Ads account with a new account](../../../../images/tutorials/create/linkedin-ads/new.png?lang=en)
+
+Next, Depending on how you want the data to be grouped you must select a *pivot* value as one of the below:
+   * ACCOUNT
+   * CAMPAIGN GROUP
+   * CAMPAIGN
+   * CREATIVE
+They will enable you to restrict export to the specified Account IDs, Campaign Group IDs or Campaign IDs. Refer to [LinkedIn Reporting & ROI API Query Parameters](https://docs.microsoft.com/en-us/linkedin/marketing/integrations/ads-reporting/ads-reporting?view=li-lms-2022-07&tabs=http#query-parameters) to understand pivoting.
+
+Finally, you must provide as a mandatory input one or more of the field values below:
+| Field | Description |
+| --- | --- |
+| accountIds | Your LinkedIn account IDs. |
+| campaignGroupIds | Your LinkedIn Campaign Group IDs. |
+| campaignIds | Your LinkedIn Campaign IDs. |
+
+> [!NOTE]
+>
+> Multiple IDs can be specified by passing comma separated values.
+
+If you fail to provide a value on selecting **[!UICONTROL Select]** you will get a `Failed to parse JSON.` or `Unknown error while processing request.` error message.
+
+![Platform UI screenshot for LinkedIn Ads showing configuration](../../../../images/tutorials/create/linkedin-ads/configuration.png?lang=en)
+
+## Next steps
+
+By following this tutorial, you have established a connection to your [!DNL LinkedIn Ads] account. You can now continue on to the next tutorial and [configure a dataflow to bring data into Platform](/help/sources/ui-tutorials/dataflow/crm.md).
+
```

Comment:
I think we need to preface this particular part about why a user needs to schedule specifically for Day or Hour? Can we add a brief intro here on why this is recommended/required?